web-api-shared/src/serde.rs

```diff
@@ -55,6 +55,48 @@ pub mod serde_duration_as_seconds {
     }
 }
 
+pub mod serde_duration_in_config {
```

Comment:
I took a glance as crate providing something like that. 

We could use some crate for this the problem is every crate a took a glance at had some issues.

Like bad code, being too flexible (we only want to have 1 way to e.g. specify 100ms, not 10 different formattings), or using a too-unusual format (e.g. partial rfc3339 duration like `PT10M20S ` it also happens to forget to define sub-seconds for duration, through a lot of code assumes they exist ), strange whitespace handling, etc.

So just dumping this minimal implementation here was faster.

E.g. `duration-string` in general does what we want, except for it's whitespace handling (ignore them so `10 00 00    s` is valid) and it's code is ... safe, correct, but not very good.